website/docs/reference/warehouse-setups/databricks-setup.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Databricks setup"
+description: ""
```

Comment:
Is a description supposed to be here?

Reply:
https://deploy-preview-3261--docs-getdbt-com.netlify.app/reference/warehouse-setups/databricks-setup

should now say: Read this guide to learn about the Databricks warehouse setup in dbt.